client/modules/core/main.js

```diff
@@ -23,6 +23,7 @@ const reactionState = new ReactiveDict();
  */
 export default {
   _shopId: new ReactiveVar(null), // The active shop
+  _shopDomain: new ReactiveVar(null), // The active shop's domain (a shop may have multiple domains)
```

Comment:
just to confirm, a `ReactiveVar` is sort of a thread-safe variable? that is, no two connection share the same instance of this?